viz_scripts/Dockerfile

```diff
@@ -1,5 +1,7 @@
 # python 3
-FROM shankari/e-mission-server:master_2024-02-10--19-38
+# Please change once all PR changes are final, so it reads from shankari/e-mission-server
+# FROM shankari/e-mission-server:master_2024-02-10--19-38
+FROM mukuflash03/e-mission-server:image-push-merge_2024-04-16--49-36
```

Comment:
Reminder that we will want to use a different account for these images in the future